live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/messaging/EventProcessor.java

```diff
@@ -49,10 +51,14 @@ public void process(Exchange exchange)
     {
         RepoEvent<?> event = eventFrom(exchange);
 
-        if (event.getType().equals(EventType.NODE_CREATED.getType()))
+        if (NODE_CREATED.getType().equals(event.getType()))
```

Comment:
We can use equality for enum values:
```suggestion
        if (event.getType() == NODE_CREATED.getType())
```

Reply:
for some reason event.getType() is string not enum😂